File: backend/GeoQuiz_backend/GeoQuiz_backend/Application/DTOs/KingOfTheHill/SubmitAnswerRequest.cs
namespace GeoQuiz_backend.Application.DTOs.KingOfTheHill
{
    public class SubmitAnswerRequest
    {
        public Guid MatchId { get; set; }
        public int RoundNumber { get; set; }
        public string CountryId { get; set; } = null!;
        public int SelectedOptionIndex { get; set; }
        public int TimeSpentMs { get; set; }
    }
}

File: frontend/app/src/main/java/com/example/geoquiz_frontend/data/remote/dtos/koth/SubmitAnswerRequest.java
package com.example.geoquiz_frontend.data.remote.dtos.koth;

public class SubmitAnswerRequest {
    private String matchId;
    private int roundNumber;
    private String countryId;
    private int selectedOptionIndex;
    private int timeSpentMs;

    public SubmitAnswerRequest(String matchId, int roundNumber, String countryId,
                               int selectedOptionIndex, int timeSpentMs) {
        this.matchId = matchId;
        this.roundNumber = roundNumber;
        this.countryId = countryId;
        this.selectedOptionIndex = selectedOptionIndex;
        this.timeSpentMs = timeSpentMs;
    }
}
